parcels/field.py

```diff
@@ -1253,47 +1225,61 @@ def __getitem__(self, key):
         raise RuntimeError('Field is in deferred_load mode, so can''t be accessed. Use .computeTimeChunk() method to force loading of  data')
 
 
-class SummedVectorField(list):
-    """Class SummedVectorField stores 2 or 3 SummedFields which defines together a vector field.
-    This enables to interpolate them as one single vector SummedField in the kernels.
+class SummedField(list):
+    """Class SummedField is a list of Fields over which Field interpolation
+    is summed. This can e.g. be used when combining multiple flow fields,
+    where the total flow is the sum of all the individual flows.
+    Note that the individual Fields can be on different Grids.
+    Also note that, since SummedFields are lists, the individual Fields can
+    still be queried through their list index (e.g. SummedField[1]).
+    SummedField is composed of either Fields or VectorFields.
 
-    :param name: Name of the vector field
-    :param U: SummedField defining the zonal component
-    :param V: SummedField defining the meridional component
-    :param W: SummedField defining the vertical component (default: None)
+    :param name: Name of the Summed field
+    :param U: List of fields. U can be a scalar Field, a VectorField, or the zonal component of the VectorField
+    :param V: List of fields defining the meridional component (default: None)
+    :param W: List of fields defining the vertical component (default: None)
```

Comment:
Somewhat clunky to explicitly call these parameters `U`, `V` and `W`, even though they could also be e.g. temperature? Why not use `Field` as first argument, and deal with the second and third using `kwargs`?

Reply:
Usually, kwargs in a function are arguments which were real arguments in another function that calls this function.
They are then neither referenced explicitly, not present in the docstring, no?

I would keep V, W explicitly. I don't really see the advantage of having a hidden kwargs.
But should we call the argument F, V, W ? And in changing this docstring 
```
    :param U: List of fields (order matters). U can be a scalar Field, a VectorField, or the zonal component of the VectorField
```
to 
```
    :param F: List of fields (order matters). F can be a scalar Field, a VectorField, or the zonal component (U) of the VectorField
```

?

Reply:
And also add
```
       :param V: List of fields defining the meridional velocity component, if F is the zonal component. (default: None)
    :param W: List of fields defining the vertical velocity component, if F and U are the zonal and meridional components (default: None)
```

Reply:
OK I agree with your argument about kwargs. 
But then I would add to the docstring for `V` and `W` that they are only used if `F` is the zonal component of a VectorField